b2b-sample/README.md

```diff
@@ -131,16 +124,23 @@ To create a user for Best Auto Mart with permissions to create an IdP for the **
         "name": "Guardio Insurance - Administrator Application",
         "tag": "Administrator Application"
       }
+    },
+    "ManagementAPIConfig": {
+      "SharedApplicationName": "Guardio-Business-App",
+      "UserStore" : "DEFAULT"
     }
   }
 }
 ```
 setup the config json file as mentioned above.
 
-### Step 5: Additional settings
+**_NOTE:_**  With the latest B2B organization management features provided by WSO2 Identity Server, the following operations are allowed in sub organization for the organization creator.
+
+1 - onboarding enterprise IDP.
+2 - Update authentication options of the shared applications.
+3 - Delete the business users.
 
-* If the federated organization id is returned in the idToken (in user_organization variable) when a user is logging
-* into an organization, remove the `CommonConfig.ApplicationConfig.SampleOrganization` section from the <i>config.json</i> file.
```

Comment:
These settings may not required to be mentioned as normal users of the b2b sample app would be the users who try the b2b organization management feature of Asgardeo. For the users who try the b2b sample app with the PCC, we can suggest in the PCC documentation for the previously released b2b sample app.

Reply:
I think it would be better if we have this part (with a change), mentioning that, in the PCC environment the developer needs to setup this to work with the sample application.